### DISCUSSION TITLE
wallet connect dashboard always pending review

### DISCUSSION THREAD

**Discussion by RuffianZhong** [WalletConnect/walletconnect-monorepo#4644] [Developer Support] [answered]
I modified the project configuration on the dashboard and submitted it for review, but it has been a week and it has been waiting for review. Please help me

**Comment by glitch-txs** [comment]
Hi, could you share your project ID? Thank you

**Reply by RuffianZhong** [reply]
> 您好，可以分享一下您的项目 ID 吗？谢谢

release = '85110a0ad9c194c61eee057b84187915'
test = 'a3dc8ad13f78f67b3b915cc23769b1fa'

**Reply by glitch-txs** [reply, marked as answer]
Hi we don't accept sumbissions from dapps any more only from wallets

**Reply by RuffianZhong** [reply]
Does this mean that no DAPPS can make payment through WalletConnect? Does WalletConnect only support authorized login?

**Reply by RuffianZhong** [reply]
But I encountered a situation where WalletConnect was unable to complete the signature, and iOS reported an invalid ID error, so I modified the configuration again

**Reply by glitch-txs** [reply]
You don't need the approval to use WalletConnect. The approval doesn't have any effects in the integration that's why we remove it :)

If you're a running an issue with the integration it must be for another reason, happy to help if you could provide more information!

**Reply by RuffianZhong** [reply]
> Hi we don't accept sumbissions from dapps any more only from wallets

So what does this mean? Can I continue to use WalletConnect to connect, sign, and initiate transactions

**Reply by glitch-txs** [reply]
Yes, of course. The approval was never a requirement to use WalletConnect, it was just for the explorer.

https://explorer.walletconnect.com/

**Reply by RuffianZhong** [reply]
> 是的，当然。使用 WalletConnect 从来不需要批准，它只是针对浏览器。
> 
> https://explorer.walletconnect.com/

I am currently using a pure JS version, and there are issues with the inability to recall signatures on the mobile web. Do you have any relevant technical documentation

**Reply by glitch-txs** [reply]
JS docs are here: https://docs.walletconnect.com/appkit/javascript/core/installation
Which SDK are you using? Which version? and which wallet?

Could you share a minimal reproducible example of the issue and steps to repro? Thanks!

**Comment by Starfeathers71** [comment]
Sorry I have been very sick I am new to all this and if it was 2nd nature I
could do in my sleep but of course I am not a clue

Thanks for fixing it I appreciate it and I hope I be back to me soon idk I
am 4th stage colon cancer and then some

**Comment by Starfeathers71** [comment]
I have adjusted what I knew how and I need to give someone else who knows
what to do the reins so I am not messing people's up when I am sick I have
choice if I am able to get on here and I am not sure what to do with a lot
I get overwhelmed I am so sorry 😔😔😔😔
